src/components/index.ts

```diff
@@ -211,7 +211,15 @@ export {ResourceItem} from './ResourceItem';
 export type {ResourceItemProps} from './ResourceItem';
 
 export {ResourceList} from './ResourceList';
-export type {ResourceListProps} from './ResourceList';
+export type {
+  ResourceListProps,
+  FilterControlProps,
```

Comment:
Maybe it makes more sense to just copy these types in web, that way we don't expand the public API of Polaris. Thoughts?